### ISSUE TITLE
For input string: "null"

### ISSUE THREAD

**Issue by marktani** [prismagraphql/graphcool-framework#553]
@kurt-o-sys commented on [Mon Aug 13 2018](https://github.com/prismagraphql/prisma/issues/2910)

**Describe the bug**
I was changing my graphql model and at a certain point, it failed to deploy. The deployment didn't succeed.
So I reverted my changes, and tried to deploy again, and I'm always getting the same error:
```
There are issues with the new service definition:

  Global
    ✖ For input string: "null"
```
When I deploy these changes on another environment, there's no problem. 
It seemed that it was due to the data, so I did a data reset on the failing environment and I tried to deploy again. Still the same error. 

In short: I can't do anything anymore on that failing environment, everything I try results in the same error (see above)

**To Reproduce**
1. run graphcool reset
2. deploy a new db model that works on another environment
3. see it doesn't deploy on this failing environment

**Expected behavior**
successful deployment, certainly after resetting the data

**Versions (please complete the following information):**
```
 cat /etc/lsb-release    
DISTRIB_ID=LinuxMint
DISTRIB_RELEASE=18
DISTRIB_CODENAME=sarah
DISTRIB_DESCRIPTION="Linux Mint 18 Sarah"
$ npm view graphcool version
...
0.11.6
...
```

**Comment by kurt-o-sys** [comment]
my projectID is: cj0wajpkpe8qh0102aoy5wed0

Is it possible to revert all changes made to the scheme? That might restore it in a functioning state. (Although the scheme as I want to deploy it, works really wel on my production environment - it's the development environment that's messed up.)

**Comment by grimunit** [comment]
I think my team found out why this happens. This was just happening to us and we realized it's because we had removed a `@defaultValue` directive from a property that had already been deployed with that directive.

So the fix was to remove the entire property first, and then redeploy with the property without the `@defaultValue` directive. Sounds like a bug in the automatic migration logic around removing `@defaultValue` directives.
